filters/ThirdParty/filter_259_Dandelion_Sprout's_Anti-Malware_List/metadata.json

```diff
@@ -0,0 +1,15 @@
+{
+  "filterId": 259,
+  "name": "Dandelion Sprout's Anti-Malware List",
+  "description": "This list goes the extra kilometer to prevent more malware than other mainstream anti-malware lists. It blocks domains and domain patterns used in malware redirection trains and in domain parking schemes, blocks sponsored Windows PUP nags on PC guide articles, uses mass blocking of domains belonging to bad IPs, and has many other subcategories that give it a solid advantage over similar lists out there.",
```

Comment:
@DandelionSprout Maybe it would be better to make the description more concise? I mean only metadata, without changing the filter itself.

How current description will look in the extension

<img width="589" height="540" alt="Image" src="https://github.com/user-attachments/assets/0115dfee-4d2a-45f1-ace1-9a630e2c2612" />